docs/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_gcs.rst

```diff
@@ -3,16 +3,107 @@
 
+    - Configured a :ref:`Data Context <tutorials__getting_started__initialize_a_data_context>`.
+    - Configured an :ref:`Expectations Suite <tutorials__getting_started__create_your_first_expectations>`.
+    - Configured a :ref:`Checkpoint <tutorials__getting_started__set_up_your_first_checkpoint>`.
+    - Configured a Google Cloud Platform (GCP) `service account <https://cloud.google.com/iam/docs/service-accounts>`_ with credentials that can access the appropriate GCP resources, which include Storage Objects.
+    - Identified the GCP project, GCS bucket, and prefix where Validations will be stored.
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+1. **Configure your GCP credentials**
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+    Check that your environment is configured with the appropriate authentication credentials needed to connect to the GCS bucket where Validations will be stored.
+
+    The Google Cloud Platform documentation describes how to verify your `authentication for the Google Cloud API <https://cloud.google.com/docs/authentication/getting-started>`_, which includes:
+
+        1. Creating a Google Cloud Platform (GCP) service account,
+        2. Setting the ``GOOGLE_APPLICATION_CREDENTIALS`` environment variable,
+        3. Verifying authentication by running a simple `Google Cloud Storage client <https://cloud.google.com/storage/docs/reference/libraries>`_ library script.
+
+2. **Identify your Data Context Validations Store**
+
+    In your ``great_expectations.yml``, look for the following lines.  The configuration tells Great Expectations to look for Validations in a store called ``validations_store``. The ``base_directory`` for ``validations_store`` is set to ``uncommitted/validations/`` by default.
+
+
+    .. code-block:: yaml
+
+        validations_store_name: validations_store
+
+        stores:
+            validations_store:
+                class_name: ValidationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: uncommitted/validations/
+
+
+3. **Update your configuration file to include a new store for Validations on GCS**
+
+    In our case, the name is set to ``validations_GCS_store``, but it can be any name you like.  We also need to make some changes to the ``store_backend`` settings.  The ``class_name`` will be set to ``TupleGCSStoreBackend``, ``project`` will be set to your GCP project, ``bucket`` will be set to the address of your GCS bucket, and ``prefix`` will be set to the folder on GCS where Validation files will be located.
+
+
+    .. warning::
+        If you are also storing :ref:`Expectations in GCS, <how_to_guides__configuring_metadata_stores__how_to_configure_an_expectation_store_in_gcs>` please ensure that the ``prefix`` values are disjoint and one is not a substring of the other.
```

Comment:
I'd recommend adding the same note about the prefix for data docs.

Reply:
good call. I'll add the corresponding note for S3 as wel